codegen/sdk-codegen/aws-models/lex-runtime-service.2016-11-28.json

```diff
@@ -1,29 +1,29 @@
 {
-  "smithy": "1.0",
+  "smithy": "1",
```

Comment:
I think smithy usually uses 1.0, any reason for changing to 1?

Reply:
This was failing if 1.0 was used, and hand to change it to 1. will go back and verify